pydantic/main.py

```diff
@@ -275,6 +275,13 @@ def model_fields_set(self) -> set[str]:
         """
         return self.__pydantic_fields_set__
 
+    @property
+    def model_computed_fields(self) -> dict[str, _decorators.Decorator[_decorators.ComputedFieldInfo]]:
```

Comment:
Could we do the following:
* Move `ComputedFieldInfo` into `pydantic.fields`
* Make `model_computed_fields` into a `dict[str, ComputedFieldInfo]`
* Store `model_computed_fields` as a class attribute on the model, like we do with 
  ```python
  # from BaseModel: ...
  model_fields: typing.ClassVar[dict[str, FieldInfo]] = {}
  ```
This feels like it would be a lot more similar to the way `model_fields` works, and I noticed in the tests below it looks like every access to an item from this dict is accessing the `info` attribute, which is of type `ComputedFieldInfo`.

If someone really needed access to the `_decorators.Decorator`, they could still access `self.__pydantic_decorators__` (of dubious publicness I guess)

Reply:
the main thing you'd want that isn't on `ComputedFieldInfo` is the property itself, which you could easily get with `getattr(model, field_name)`.

I think that's a good idea, the only change I might propose (and we could definitely make this change if/when it's needed), is to store them both in dunder methods `__pydantic_fields__` and `__pydantic_computed_fields__`, then expose them via a property, so they can work on `BareModel` too if we get there.

Reply:
That all sounds good to me.

Reply:
done, but kept `model_computed_fields` as a property.